awx/main/dispatch/control.py

```diff
@@ -21,7 +24,9 @@ def __init__(self, service, host=None):
         self.queuename = host or get_local_queuename()
 
     def status(self, *args, **kwargs):
-        return self.control_with_reply('status', *args, **kwargs)
+        r = redis.Redis.from_url(settings.BROKER_URL)
+        stats = r.get(f'awx_{self.service}_statistics') or b''
```

Comment:
Each service (callback, dispatcher) has its own specific key in redis where the service periodically does a ` .put()` containing a view of internal state.

Reply:
This model more closely matches the behavior in the new broadcast stats support @chrismeyersfsu added (and I like it better, because it means that as long as redis is running, we can still see the "last stats" even if one of these processes is having trouble).